models/articles.model.js

```diff
@@ -37,4 +37,20 @@ function selectArticles(topic, author, sort_by = "created_at", order = "DESC") {
     })
 }
 
-module.exports = {selectArticleById, selectArticles}
\ No newline at end of file
+function updatedArticlebyId(article_id, body) {
+    const bodyKeys = Object.keys(body);
+    if (bodyKeys.length !== 1 || !bodyKeys.includes('inc_votes')) {
```

Comment:
Great work! But I would argue that it should simply ignore unnecessary properties, as there is no need to reject as long as there is still an 'inc votes' key